astrobase/varbase/lcfit.py

```diff
@@ -123,6 +126,14 @@ def LOGEXCEPTION(message):
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 
+try:
```

Comment:
It's a mixed bag, because it raises computational cost by **a lot**. I think provided it's optional, it's ok. But I feel like it's best for use cases where you really do care about the errors.

Reply:
Fair enough. I'll make it optional (or down the line, move the emcee enabled stuff into another lcfit module).